### PR TITLE
Added avatar/profile picture link to the info command. 

### PR DESCRIPTION
**Pull request by WitherredAway**
Added a field 'Avatar' which shows the link of the `user`'s profile picture.

### CODE REVIEW

**Comment by jacobmonck** [comment]
Haven't been in the discord server, but is this picture in the embed thumbnail. Because if not I would just put their profile picture in the embed thumbnail and they can just get the link from that.

```py
embed.set_thumbnail(url=user.avatar_url)
```

**Comment by WitherredAway** [comment]
yes it is in the embed thumbnail, but you can't copy from that on mobile.

File: cogs/bot.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ async def info(self, ctx, *, user: Union[discord.Member, FetchUserConverter] = N
         embed.set_author(name=str(user))
 
         embed.add_field(name="ID", value=user.id, inline=False)
+        embed.add_field(name="Avatar", value=f"[Link]({user.avatar_url})", inline=False)
```
Comment (oliver-ni) [inline cogs/bot.py]: This does not work on discord.py 2.0. [see docs](https://discordpy.readthedocs.io/en/master/api.html#discord.User.avatar)

**Review by oliver-ni — CHANGES_REQUESTED** [review]

File: cogs/bot.py (hunk 1 above)
Comment (oliver-ni) [reply cogs/bot.py]:
```suggestion
        embed.add_field(name="Avatar", value=f"[Link]({user.avatar.url})", inline=False)
```

**Review by oliver-ni — CHANGES_REQUESTED** [review]

**Comment by WitherredAway** [comment]
ah I see, thank you for letting me know

**Pull request merged**